Frameworks/include/CoreTextInternal.h

```diff
@@ -88,6 +73,61 @@ struct __CTFrame : CoreFoundation::CppBase<__CTFrame> {
 
 #pragma endregion CTFrame
 
+#pragma region CTRun
+struct __CTRun : CoreFoundation::CppBase<__CTRun> {
+    __CTRun()
+        : _attributes(CFDictionaryCreateMutable(kCFAllocatorSystemDefault,
+                                                0,
+                                                &kCFCopyStringDictionaryKeyCallBacks,
+                                                &kCFTypeDictionaryValueCallBacks)),
+          _textMatrix(CGAffineTransformIdentity) {
+    }
+
+    __CTRun(const CTRunRef& other);
+
+    ~__CTRun() {
+        _SafeRelease(&_dwriteGlyphRun.fontFace);
+        delete[] _dwriteGlyphRun.glyphIndices;
+        delete[] _dwriteGlyphRun.glyphAdvances;
+        delete[] _dwriteGlyphRun.glyphOffsets;
+    }
+
+    static CFTypeID GetTypeID() {
```

Comment:
The point of CppBase is that you do not have to think about this and the only reason to override it is to set different parameters.
Are we doing that?